modules/subdyn/src/SubDyn.f90

```diff
@@ -1152,13 +1152,45 @@ SUBROUTINE SD_Input(SDInputFile, Init, p, ErrStat,ErrMsg)
 if (LegacyFormat) then
```

Comment:
NOTE: the LegacyFormat has been here for a couple of release, it's likely time to remove it. Now we can introduce a new legacy format regarding your new changes...

Reply:
Agreed

Reply:
@ebranlard It was also problematic because LegacyFormat was previously inferred from the joint parsing rather than the member parsing. In this instance, the member type will have been excluded and assumed to be one even if the user desired a rigid or cable.